orchagent/vxlanorch.cpp

```diff
@@ -548,6 +548,13 @@ bool VxlanTunnelOrch::delOperation(const Request& request)
         return true;
     }
 
+    VxlanTunnelMapOrch* tunnel_map_orch = gDirectory.get<VxlanTunnelMapOrch*>();
+    if (tunnel_map_orch->anyMapForTunnel(tunnel_name))
```

Comment:
returning false will mean the orchagent continues to process this event till the map is removed. Is it not better to 
do this check and return false in the vxlanmgr ?